### DISCUSSION TITLE
Python 3.7 spaCy Help Needed- Environment Inconsistency Issue?

### DISCUSSION THREAD

**Discussion by melissafeeney** [explosion/spaCy#7795] [Help: Coding & Implementations] [answered]
Help needed please!

I am facing an issue when trying to call spaCy into my Jupyter notebook. When I run `import spacy` I get the below:
<img width="974" alt="Screen Shot 2021-04-14 at 7 28 59 PM" src="https://user-images.githubusercontent.com/31778500/114868673-2e3f8e00-9dc4-11eb-9f58-d8a813cae4e2.png">

I have used spaCy before many times with no issue, but I noticed this problem began after I was trying to also install from neuralcoref import Coref and am not sure if that has caused this.

When I go into the terminal and run `conda list spacy `it looks like spaCy is available:
![Screen Shot 2021-04-14 at 7 33 47 PM](https://user-images.githubusercontent.com/31778500/114868741-41eaf480-9dc4-11eb-929d-6d0cc7bfdb96.png)

I am really struggling with this issue, but I tried to reinstall murmurhash using `conda install -c anaconda murmurhash` after which I got this. This is just a screenshot of the first few but there are MANY packages that are allegedly causing the inconsistency:
![Screen Shot 2021-04-14 at 7 50 59 PM](https://user-images.githubusercontent.com/31778500/114868891-6941c180-9dc4-11eb-9afa-702d20ea72ca.png)

Following the list of packages causing inconsistencies, I get this:
![Screen Shot 2021-04-14 at 7 52 12 PM](https://user-images.githubusercontent.com/31778500/114868973-7bbbfb00-9dc4-11eb-9aff-4ca5c8100a69.png)

For reference, I am using MacOS and python 3.7. How can I fix this? Thank you in advance!

**Comment by polm** [comment, marked as answer]
Hard to tell exactly what's going on with your environment, but one issue is that neuralcoref isn't compatible with spaCy 3. 

Could you create a new blank environment and install spaCy there?

**Reply by polm** [reply]
Looks like this was indeed due to neuralcoref not working with spaCy v3. 

https://stackoverflow.com/questions/67100601/python-3-7-spacy-help-needed-environment-inconsistency-issue
